t/user.t

```diff
@@ -41,8 +41,13 @@ subtest 'password' => sub {
   eval { $user->set_password('') };
   like $@, qr{Usage:.*plain}, 'set_password() require plain string';
   ok !$user->password, 'no password';
+
+  $user->{password} = '$2a$08$0KlK5QzJjzWFNW6JsuT52..GXe1sTRWZU1es8hfo0HcD29tTFzvsi';
+  $user->validate_password('s3cret', 'validate password backwards compatible with bcrypt');
```

Comment:
❤️

Reply:
This test was broken, woopsie, so fixed it.